src/panels/lovelace/hui-root.ts

```diff
@@ -7,7 +7,6 @@ import {
   mdiDotsVertical,
   mdiFileMultiple,
   mdiFormatListBulletedTriangle,
-  mdiHelp,
```

Comment:
This icon was tied to an unreachable list item, which I believe was inside a `this._editMode` ternary section, which was itself inside another `!this._editMode` ternary section. I believe it was redundant.